docs/source/tutorials_wine.rst

```diff
@@ -0,0 +1,590 @@
+- ``legionctl`` is installed locally or Jupyter Lab with legion plugin is installed locally or in the cloud.
+
+Pre-sequences
+~~~~~~~~~~~~~~~~~~~
+
+To train, pack and deploy model you need to interact with Legion Platform EDI server. This server provide REST api.
+You can use it directly or using different tools.
+
+In this tutorial we will describe the process of interacting with EDI server using ``legionctl`` command line tool and
+Jupyter Lab with Legion Platform plugin.
+
+In the process of model training/packing/deployment Legion Platform should work with different kinds of data that you should provide.
+They are: source code of the train script, input data and so on.
+
+Legion Platform uses concept of ``Connections`` to manage all kinds of such data. Read more about ``Connections``.
+
+
+Authentication
+##################
+To interact with Legion Platform you should be authenticated via Keycloak. Read more about all available auth methods.
+
+`Using legionctl`
+
+Use interactive login mode for authenticate with Legion Platform
+Just type in terminal next command:
+
+.. code-block:: console
+
+    $ legionctl login --edi $EDI_URL
+
+Where ``$EDI_URL`` is host of edi server. Usually it has address ``https://edi.<legion-platform-host>``
+
+After running command above interactive session will be run and you will be invited to open the link that appeared in
+terminal.
+
+Click to the link and keycloak UI will be opened in your web browser. Two cases are possible:
+
+– You are already authenticated at keycloak. In this case you will see series of redirects and the result web page
+with suggestion to close this window already. Just return to terminal and you will see ``Success! Credentials have been saved.``
+
+– You have not been authenticated at keycloak yet. In this case you will see keycloak UI and should press ``Google`` button.
+After this just login in your EPAM account (after entering EPAM email you will be redirected yo EPAM login page).
+Then you can close the browser tab. Just return to the terminal and you will see ``Success! Credentials have been saved.``
+
+After successful authentication access token will be saved in your config. This token will be used by ``legionctl`` for all
+API calls to the EDI server under the hood:
+
+.. code-block:: console
+
+    $ legionctl config get edi_access_token --show-secrets
+
+`Using jupyterlab`
+
+The method of authentication depends on how to your Jupyter Lab is configured. Read more about different auth methods
+with Jupyter Lab plugin.
+
+Prepare code and data
+######################
+
+Prepare MLFlow project
+------------------------
+
+In this tutorial we will solve the famous `hello world` problem of wine classification using Legion Platform with MLFlow integration.
+
+First of all let's review project source code.
+To to this you should clone legion-examples github repository. This repo includes examples of projects that are ready to be used
```

Comment:
I guess it should be described in way like :
"in order to do it, we create a simple model, then we add MLProject file, then ..., then .... 
Final files can be viewed by next link..."